oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,202 @@
+
+			-- `OpenShift docs <https://docs.openshift.com/container-platform/4.8/architecture/architecture-rhcos.html#rhcos-about_architecture-rhcos>`_
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on OpenShift.
+
+	.. tab:: Tanzu
+
+		See the `Tanzu guide <https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html>`_
+		to learn how to create a cluster.
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on Tanzu.
+
+	.. tab:: Nutanix
+
+		See the `Nutanix Karbon Guide <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-deploy-cluster-c.html>`_
+		to learn how to create a cluster.
+
+
+		**Limitations**
+
+		Nutanix does not have Ubuntu Pro offerings for the nodes, i.e. you cannot
+		choose Ubuntu Pro images for the nodes.
+
+			"Deploying Kubernetes clusters in Karbon requires a **CentOS** image.
+			You must choose from a CentOS version and download the image."
+
+			-- `Nutanix docs: "Downloading Images" <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-upload-image-t.html>`_
+
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on Nutanix.
+
+
+Create a Secret for Private Registry
+------------------------------------
+
+If you published your image to a private registry, you will need to create a
+`secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_ in Kubernetes.
+`See here <https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/>`_
+to learn about pulling images from private registries.
+
+You can use the following example command to create a secret named ``regcred`` for
+`Docker Hub <https://hub.docker.com>`_. For this, use ``https://index.docker.io/v1/``
```

Comment:
```suggestion
`Docker Hub <https://hub.docker.com>`_ (i.e. using ``https://index.docker.io/v1/``
```

Reply:
is it https://index.docker.io/v1/ or just docker.io?

Reply:
For kubernetes docs it's with the index